White_Chocolate_Espresso.swift

```diff
@@ -1,2 +1,8 @@
 // Hippe neue Variante von Objective-C
-print("White Chocolate Espresso - 1,00€")
+
+func nameWith(price: String) -> String {
```

Comment:
what protocol does the `price` extend here?

Reply:
is the PWT correct?

Reply:
does this compile on swiftc 4?

Reply:
what Xcode version do I need?

Reply:
does this require foundation

Reply:
couldn't you do this with use of a coffee-machine-observer-observer pattern instead?

Reply:
You troll :D This is Swift 4 (and probably earlier versions as well), requires neither Xcode nor Foundation and could possibly do with a lot more observer patterns or RxSwift :P